packages/frontend/web/components/Outbrain.tsx

```diff
@@ -49,6 +53,60 @@ const OutbrainWidget: React.FC<{}> = ({}) => {
     );
 };
 
+const outbrainContainer = css`
+    .js-outbrain {
+        width: 100vw;
+        position: relative;
+        left: 50%;
+        right: 50%;
+        margin-left: -50vw;
+        margin-right: -50vw;
+        background-color: ${palette.neutral[97]};
+    }
+
+    .js-outbrain-container {
+        margin: auto;
+        border-left: 1px solid ${palette.neutral[86]};
+        border-right: 1px solid ${palette.neutral[86]};
+        border-top: 1px solid ${palette.neutral[86]};
+        padding: 20px;
+
+        ${tablet} {
```

Comment:
How come we need a max width on this, does it break out? I thought as it's full width it should just need a 100% width

Reply:
oh maybe because the `vw` set above don't include scroll bar (ergh) so the max width gets past that?

Reply:
Yeah, it was breaking out of the parent container and auto setting a margin. I had to include the parent container in order to be able to replicate the html structure on frontend.